backend/dal/Security/Permissions.cs

```diff
@@ -70,6 +70,9 @@ public enum Permissions
         ReportsView = 34,
 
         [Display(GroupName = "project", Name = "reports-spl", Description = "Can view/modify/generated project spl reports.")]
```

Comment:
Might want to update the description to match what you have in the above description